sections/main-product.liquid

```diff
@@ -308,9 +309,9 @@
               %}
 
             {%- when 'variant_picker' -%}
-              {% render 'product-variant-picker', block: block, product: product %}
+              {% render 'product-variant-picker', product: product, block: block, product_form_id: product_form_id %}
             {%- when 'buy_buttons' -%}
-              {%- render 'buy-buttons', block: block, product: product, product_form_id: product_form_id, show_pickup_availability: true -%}
+              {%- render 'buy-buttons', block: block, product: product, product_form_id: product_form_id, section_id: section.id, show_pickup_availability: true -%}
             {%- when 'rating' -%}
               {%- if product.metafields.reviews.rating.value != blank -%}
                 {% liquid
```

Comment:
**Can be addressed in a smaller, separate PR. Not a blocker to ship this PR, possible blocker for release.**

Not related to this line, just didn't have anywhere else to leave this suggestion.

It seems like Quick Buy styles are broken for products with no image:
![image](https://user-images.githubusercontent.com/37168033/216172883-d6374e13-6948-4779-aef3-cb9bd18a286c.png)

Expected:
![image](https://user-images.githubusercontent.com/37168033/216173086-99b836b8-650b-4d6f-bb25-74c6b86233bd.png)

It seems like the `max-width: 100%;` is taking over the `max-width: 57rem`, making the area full width and not centered.

Reply:
Issue created - https://github.com/Shopify/dawn/issues/2274